draft-ietf-tls-dtls-connection-id.md

```diff
@@ -132,8 +132,8 @@ the server to use a zero-length CID; the result is the same).
   } ConnectionId;
 ~~~~
 
-A server which is willing to use CIDs will respond with its own
-"connection_id" extension, containing the CID which it wishes the
+A server that is willing to use CIDs will respond with its own
```

Comment:
Please don't do this. http://itre.cis.upenn.edu/~myl/languagelog/archives/001464.html

Reply:
ACK